Description: disable running configure
 since dh_autoreconf won't let us pass --without-gimp to this.
 (Debian Bug #623229
Index: gimp-help-2.6.1/autogen.sh
===================================================================
--- gimp-help-2.6.1.orig/autogen.sh
+++ gimp-help-2.6.1/autogen.sh
@@ -58,18 +58,18 @@ test $TEST_TYPE $FILE || {
 }
 
 
-echo
-echo "I am going to run ./configure with the following arguments:"
-echo
-echo "  --enable-maintainer-mode --enable-build $AUTOGEN_CONFIGURE_ARGS $@"
-echo
-
-if test -z "$*"; then
-    echo "If you wish to pass additional arguments, please specify them "
-    echo "on the $0 command line or set the AUTOGEN_CONFIGURE_ARGS "
-    echo "environment variable."
-    echo
-fi
+#echo
+#echo "I am going to run ./configure with the following arguments:"
+#echo
+#echo "  --enable-maintainer-mode --enable-build $AUTOGEN_CONFIGURE_ARGS $@"
+#echo
+#
+#if test -z "$*"; then
+#    echo "If you wish to pass additional arguments, please specify them "
+#    echo "on the $0 command line or set the AUTOGEN_CONFIGURE_ARGS "
+#    echo "environment variable."
+#    echo
+#fi
 
 if test -z "$ACLOCAL_FLAGS"; then
     acdir=`$ACLOCAL --print-ac-dir`
@@ -96,7 +96,7 @@ if test $RC -ne 0; then
    exit 1
 fi
 
-$AUTOMAKE --add-missing || exit 1
+$AUTOMAKE --add-missing --copy || exit 1
 if [ -e Makefile.in ]; then
     sed -e 's/^# HIDE FROM AUTOMAKE #//' \
         -e '/^all\(-local\)\?:/i\
@@ -114,11 +114,11 @@ rm -rf autom4te.cache
 
 cd $ORIGDIR
 
-if $srcdir/configure --enable-maintainer-mode "$@"; then
-  echo
-  echo "Now type 'make' to compile $PROJECT."
-else
-  echo
-  echo "Configure failed or did not finish!"
-  exit 1
-fi
+#if $srcdir/configure --enable-maintainer-mode "$@"; then
+#  echo
+#  echo "Now type 'make' to compile $PROJECT."
+#else
+#  echo
+#  echo "Configure failed or did not finish!"
+#  exit 1
+#fi
